src/lib/components/streams/streams-manager.svelte

```diff
@@ -54,6 +56,12 @@
             color: 'dark',
         },
     ]
+    export let streamsFilter: FilterCheckbox[] = [
+        {
+            label: 'Only own channels',
+            onChange: onOnlyOwnChannels,
+        },
```

Comment:
please add the value here:
```
    export let streamsFilter: FilterCheckbox[] = [
        {
            label: 'Only own channels',
            onChange: onOnlyOwnChannels,
            value: get(authenticatedUserDID),
        },
    ]
```

Reply:
I changed your suggestion to
`value: get(identityFilterOptions).creatorFilterState ? get(authenticatedUserDID) : undefined,`
because with the current logic the value is related to the state of the checkbox, if the value is always set to the userDID the checkbox will always be checked after switching e.g. from the Identities tab to the Steams tab